pkg/operation/seed/seed.go

```diff
@@ -172,22 +172,19 @@ func generateWantedSecrets(seed *Seed, certificateAuthorities map[string]*utilse
 		)
 	}
 
-	// VPA feature gate
-	if controllermanagerfeatures.FeatureGate.Enabled(features.VPA) {
-		secretList = append(secretList,
-			&utilsecrets.CertificateSecretConfig{
-				Name: "vpa-tls-certs",
+	secretList = append(secretList,
```

Comment:
Can you add this directly to the above `secretList` instead of using `append()` here?